.github/workflows/delete-remote-pins.yaml

```diff
@@ -0,0 +1,44 @@
+name: Delete Remote Pins
+
+on:
+  workflow_dispatch:
+    inputs:
+        batch_size:
+          required: false
+          description: Size of a delete batch
+          type: number
+
+jobs:
+  delete-remote-pins:
+    name: Delete remote pins
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        env: [
+          'staging', 
+          # 'production' Enable once tested in staging
```

Comment:
Disabled Prod for now.
Let's enable once we tested this on staging.

Reply:
Maybe we can use env as input to not need to change this for running prod?